onnxruntime/core/providers/js/operators/resize.cc

```diff
@@ -45,12 +45,12 @@ namespace js {
       Resize);
 
 #define REGISTER_RESIZE_KERNEL_DOMAIN(domain)       \
-  REGISTER_RESIZE_VERSIONED_10_10_KERNEL(domain);   \
   REGISTER_RESIZE_VERSIONED_KERNEL(domain, 11, 12); \
   REGISTER_RESIZE_VERSIONED_KERNEL(domain, 13, 17); \
   REGISTER_RESIZE_VERSIONED_KERNEL(domain, 18, 18); \
   REGISTER_RESIZE_KERNEL(domain, 19);
 
+REGISTER_RESIZE_VERSIONED_10_10_KERNEL(kOnnxDomain);
```

Comment:
What is unique about opset version 11? Because we support opset 10 in ONNX domain does it make sense to add missing schema for opset 10 for NHWC?